objectivec/DevTools/full_mac_build.sh

```diff
@@ -224,28 +224,10 @@ if [[ "${DO_XCODE_IOS_TESTS}" == "yes" ]] ; then
   # just pick a mix of OS Versions and 32/64 bit.
   # NOTE: Different Xcode have different simulated hardware/os support.
   case "${XCODE_VERSION}" in
-    [6-8].* )
-      echo "ERROR: The unittests include Swift code that is now Swift 4.0." 1>&2
-      echo "ERROR: Xcode 9.0 or higher is required to build the test suite, but the library works with Xcode 7.x." 1>&2
+    [6-9].* )
+      echo "ERROR: Xcode 9.0 or higher is required." 1>&2
```

Comment:
Don't we want this to be 10.2 or better? Can we push it up to higher than that (given that Apple won't accept anything iOS with less than 13 AFAIK)

Reply:
Fixed message.

I'll look at cycling back to raising this in the future. But for now, just want to keep it as is, but prune some of the stale things for where the mins were.